pkgs/development/python-modules/py-multicodec/default.nix

```diff
@@ -0,0 +1,54 @@
+{ pkgs
+, buildPythonPackage
+, fetchFromGitHub
+, pytest-runner
+, pytestCheckHook
+, pythonOlder
+, morphys
+, six
+, varint
+}:
+
+buildPythonPackage rec {
+  pname = "py-multicodec";
+  version = "0.2.1";
+  disabled = pythonOlder "3.5";
+
+  src = fetchFromGitHub {
+    owner = "multiformats";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-2aK+bfhqCMqSO+mtrHIfNQmQpQHpwd7yHseI/3O7Sp4=";
+  };
+
+  patches = [
+    # Error without this patch:
+    # Failed: [pytest] section in setup.cfg files is no longer supported, change to [tool:pytest] instead.
+    ./pytest.patch
+  ];
+
+  dontConfigure = true;
```

Comment:
Why do want this?

Reply:
It reduces the build output by two lines:
```
configuring
no configure script, doing nothing
```
But I don't really care either way, so I removed `dontConfigure` again.